open-metadata-resources/open-metadata-labs/administration-labs/understanding-platform-services.ipynb

```diff
@@ -110,7 +110,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "platformServicesURLRoot = platformURLroot + \"/open-metadata/platform-services/users/\" + adminUserId + \"/server-platform\""
+    "platformServicesURLRoot = serverPlatformURL + \"/open-metadata/platform-services/users/\" + adminUserId + \"/server-platform\""
```

Comment:
I am not sure why you want to change `platformURLroot` to `serverPlatformURL`. we want to be very clear what is platform and what is server so I think we should avoid serverplatform

Reply:
As explained earlier:
the functions queryKnownServers and queryActiveServers accept as 2nd parameter a url root.
In our original notebook we called the functions with parameter platformServicesURLRoot which was wrong.

In the functions queryKnownServers and queryActiveServers the 2nd parameter is inside the functions called serverPlatformURL.
The 2nd parameter should contain the url + portnumber.
Then inside the functions based on the parameter it will generate a variable called platformServicesURLRoot.

I renamed platformURLroot to serverPlatformURL to align with the name used internally in the function.
Inside the functions the following statement is used
platformServicesURLRoot = serverPlatformURL + "/open-metadata/platform-services/users/" + adminUserId + "/server-platform"
So the statement in the notebook is then also identical to the statement as used in the common functions.

Reply:
I think I see your point now. 
In the notebook you want to see it is related to a platform, while in the function call internally you want to return a server so there it makes sense to use server in the name.
I will change it back

Reply:
@davidradl just saw your comment on getting consensus.
I will wait with reverting till it is agreed what we are going to use.

Reply:
I have reverted the renaming as discussed.